src/globus_cli/login_manager/__init__.py

```diff
@@ -15,6 +15,7 @@
 
 __all__ = [
     "MissingLoginError",
+    "compute_timer_scope",
```

Comment:
```suggestion
```

This is an artifact of the rebase which I missed. It's included at the bottom of the list.